remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,34 @@
+package remoteworkitem
```

Comment:
Yes that works @aslakknutsen . But I'm unsure about why we should be doing this since I don't see a different package name for tests in other parts of the code.